onecodex/viz/_metadata.py

```diff
@@ -229,27 +211,29 @@ def plot_metadata(
             if sort_x:
                 raise OneCodexException("Must not specify sort_x when plot_type is boxplot")
 
-            box_size = 45
-            increment = 5
-            n_boxes = len(df[magic_fields[haxis]].unique())
+            boxplot_kwargs = {"median": {"stroke": "black"}}
+            if not secondary_haxis or width == "container":
```

Comment:
Can you add a type annotation for `width: (int | Literal["container"])`

Reply:
Added type annotations and docstrings for `width` and `height` params